_redirects

```diff
@@ -13,6 +13,7 @@
 /api                /docs/tools/api
 /cli                /docs/tools/cli.html
 /terraform          /docs/tools/terraform
+/community          https://aiven.io/community/
```

Comment:
Hi Floor 👋 

If we're gonna have a redirect, let's delete the page altogether that way we do some winter cleaning 😊  Could you delete the page and update the links to the old location to the new one if you don't mind?